client/webserver/site/src/html/dexsettings.tmpl

```diff
@@ -46,24 +46,44 @@
       {{template "dexAddrForm" .}}
     </form>
 
-    {{- /* UPDATE BOND OPTIONS */ -}}
-    <form class="d-hide" id="updateBondOptionsForm" autocomplete="off">
-      <div class="py-1 text-center fs28 sans-light">[[[update_bond_options]]]</div>
-      <div class="selectBondAsset mt-3">
+    {{- /* BOND DETAILS */ -}}
+    <form class="d-hide" id="bondDetailsForm" autocomplete="off">
+      <div class="form-closer hoverbg"><span class="ico-cross"></span></div>
+      <div class="py-1 fs28 text-center sans-light">[[[bond_details]]]</div>
+      <div class="d-flex justify-content-between my-3">
+        <span class="text-center">
+          [[[current_tier]]]
+          <span class="ico-info" data-tooltip="[[[current_tier_tooltip]]]"></span>
+          <br>
+          <span id="currentTier">10</span>
+        </span>
+        <span class="text-center"> 
+          [[[pending_bonds]]]
+          <span class="ico-info" data-tooltip="[[[pending_bonds_tooltip]]]"></span>
+          <br>
+          <span id="pendingBonds">10</span>
+        </span>
+        <span class="text-center">
+          [[[expired_bonds]]]
+          <span class="ico-info" data-tooltip="[[[expired_bonds_tooltip]]]"></span>
+          <br>
+          <span id="expiredBonds">10</span>
+        </span>
```

Comment:
I don't think a normal user would find this information useful. It risks confusing people. How about we start with a table of some sort, showing, for all bond assets, 
1. Cost of a single bond, with fiat conversion if available
2. Total locked amount for single bond, with fiat conversion if available
3. Current target tier and server reported tier

We could also provide some basic info about how trading limits will scale with bond tier, and that additional bonds can offset penalties.

Reply:
Sure, we can provide additional information on this page. I recommend displaying information for one asset at a time. Users can easily choose another asset, and we will then display its corresponding information.

Reply:
After I bond for 1 tier, I want to know how much is going to be unavailable to me once I bond. Something like `bondAmt * targetTier * 2 + bondsFeeBuffer`.

Reply:
> After I bond for 1 tier, I want to know how much is going to be unavailable. Something like `bondAmt * targetTier * 2 + bondsFeeBuffer`.

Oh, this. Okay.